internal/db/db_users_test.go

```diff
@@ -0,0 +1,72 @@
+package db_test
+
+import (
+	"testing"
+
+	"github.com/canonical/notary/internal/db"
+	"golang.org/x/crypto/bcrypt"
+)
+
+func TestUsersEndToEnd(t *testing.T) {
```

Comment:
While I think those tests have just been refactored to their own files, I think they only cover the happy path and do not test the error cases. I would be fine with creating a separate task to fix this in a separate PR.

Reply:
Since the users aren't relevant to this PR, i'll create a new ticket for them but i'll add more tests for the certificates.